mautic/mautic
# fixed direction being lost on pagination, contact link field being lo…

…st on pagination, made Email for SF required

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When using a CRM plugin form the direction of the data flow would get lost when changing pages, the Contact's timeline link field would also get lost and for Salesforce plugin, the Email was made required
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Open a CRM plugin config form
2. navigate through the pages, see previously direction configuration get lost
3. navigate through pages and see the Mautic contact's timeline link field get lost
4. Email field for SF was not required

#### Steps to test this PR:
1. Apply PR
2. navigate through pages and direction configuration should be set on all pages for matched fields.
3. Timeline link field should not disappear
4. Email for Salesforce is now required
